content/en/functions/collections/D.md

```diff
@@ -0,0 +1,49 @@
+---
+title: collections.D
+description: Returns a slice of sequentially ordered random integers.
+categories: []
+keywords: [random]
+params:
+  functions_and_methods:
+    returnType: '[]int'
+    signatures: [collections.D SEED N MAX]
```

Comment:
I have renamed max => hi in the implementation; hi is commonly used as the upper boundary in Go slice notation, which should give a slightly better indication of the half-open interval just by looking at the name.

Reply:
The signature is now `collections.D SEED N HIGH`.